test/functional/services/dashboard/add_panel.js

```diff
@@ -16,31 +16,51 @@ export function DashboardAddPanelProvider({ getService, getPageObjects }) {
       await testSubjects.click('addNewSavedObjectLink');
     }
 
-    async closeAddVizualizationPanel() {
-      log.debug('closeAddVizualizationPanel');
-      await find.clickByCssSelector('i.fa fa-chevron-up');
-    }
-
     async clickSavedSearchTab() {
       await testSubjects.click('addSavedSearchTab');
     }
 
     async addEveryEmbeddableOnCurrentPage() {
       log.debug('addEveryEmbeddableOnCurrentPage');
-      const embeddableRows = await find.allByCssSelector('.list-group-menu-item');
+      const addPanel = await testSubjects.find('dashboardAddPanel');
+      const embeddableRows = await addPanel.findAllByClassName('euiLink');
       for (let i = 0; i < embeddableRows.length; i++) {
         await embeddableRows[i].click();
       }
       log.debug(`Added ${embeddableRows.length} embeddables`);
     }
 
     async clickPagerNextButton() {
-      const pagerNextButtonExists = await testSubjects.exists('paginateNext');
-      if (pagerNextButtonExists) {
-        await testSubjects.click('paginateNext');
-        await PageObjects.header.waitUntilLoadingHasFinished();
+      // Clear all toasts that could hide pagination controls
+      const toasts = await find.allByCssSelector('.euiToast');
```

Comment:
I think this will be very useful logic for other tests too.  Maybe put into PageObjects.header? or PageObjects.common?  Or add a new toastsService?